out/production/MicroBlog/templates/messages.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>test</title>
+</head>
+<body>
+<form action="/Submit" method="post">
+    <input type="text" placeholder="Write a message" name="Message"/>
+    <button type="submit">Submit</button>
+
+<ol>
+    {{#post}}
+    <li>{{post}}</li>
+    {{/post}}
```

Comment:
Nice job Vegas.  Just one thing, in your messages.html the mustache syntax is a bit off.  The list item you want to display is not post which is the entire arraylist.  It should be {{.}} which will display each string.  Otherwise good job.